applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -268,16 +266,14 @@ bool verify_datamode_control(uint16_t time_limit, uint16_t *min_time_limit)
 	return true;
 }
 
-static void response_handler(void *context, const char *response)
-{
-	int len = strlen(response);
-
-	ARG_UNUSED(context);
+AT_MONITOR(at_notify, ANY, notification_handler);
 
-	/* Forward the data over UART */
-	if (slm_operation_mode == SLM_AT_COMMAND_MODE && len > 0) {
+static void notification_handler(const char *response)
+{
+	if (slm_operation_mode == SLM_AT_COMMAND_MODE && strlen(response) > 0) {
```

Comment:
I don't think there is a chance a notification might be empty, so probably you can avoid the `strlen` check.

Reply:
If you say so 😀 ... removed